go/externalstest/test_common.go

```diff
@@ -10,13 +10,36 @@ import (
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/pvlsource"
 	"github.com/keybase/client/go/uidmap"
+	insecureTriplesec "github.com/keybase/go-triplesec-insecure"
 )
 
-func SetupTest(tb libkb.TestingTB, name string, depth int) (tc libkb.TestContext) {
-	ret := libkb.SetupTest(tb, name, depth+1)
+// SetupTest ignores the third argument.
+func SetupTest(tb libkb.TestingTB, name string, depthIgnored int) (tc libkb.TestContext) {
+	// libkb.SetupTest ignores the third argument (depth).
+	ret := libkb.SetupTest(tb, name, depthIgnored)
 
 	ret.G.SetServices(externals.GetServices())
 	ret.G.SetUIDMapper(uidmap.NewUIDMap(10000))
 	pvlsource.NewPvlSourceAndInstall(ret.G)
 	return ret
 }
+
+func InstallInsecureTriplesec(g *libkb.GlobalContext) {
```

Comment:
We can't do this.  I don't want this function to appear in a non-`_test.go` file, since it's so insanely dangerous.  We need to keep it out of the production build at all costs, with belt and suspenders!